scipy/optimize/_nnls.py

```diff
@@ -1,10 +1,11 @@
-from . import __nnls
-from numpy import asarray_chkfinite, zeros, double
+import numpy as np
+from scipy.linalg import solve
+
 
 __all__ = ['nnls']
 
 
-def nnls(A, b, maxiter=None):
+def nnls(A, b, maxiter=None, atol=None):
```

Comment:
Maybe we should consider?
```suggestion
def nnls(A, b, maxiter=None, *, atol=None):
```

Reply:
I think you are right, we should probably do this more often.